connector/ldap/ldap.go

```diff
@@ -50,6 +50,11 @@ import (
 //         groupAttr: member
 //         nameAttr: name
 //
+//         #Used when the filter member:1.2.840.113556.1.4.1941: timesout
+//         #Better implementation of recursive search
```

Comment:
Maybe we can refer to this as "recursive" instead of "inheritance"

Reply:
Done